deployment/GP03FLMA_Deploy.csv

```diff
@@ -89,21 +89,39 @@ SR17-10,,SR1811,,GP03FLMA-RIS01-00-SIOENG000,5,1,2017-07-13T23:00:00,2018-07-27T
+SR1811,,,,GP03FLMA-RIM01-02-CTDMOH049,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-CTDMOH-13653,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIM01-02-CTDMOH050,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-CTDMOH-13444,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIM01-02-CTDMOH051,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-CTDMOH-11704,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIS01-00-SIOENG000,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGCON-SIOCON-13279,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIS01-03-DOSTAD000,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-DOSTAD-00228,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIS01-04-PHSENF000,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-PHSENF-P0143,49.976145,-144.246335,,,4123,
+SR1811,,,,GP03FLMA-RIS01-05-FLORTD000,6,1,2018-07-23T22:54:00,,CGMGP-03FLMA-00006,,CGINS-FLORTD-01321,49.976145,-144.246335,,,4123,
+SKQ201920S,,,,GP03FLMA-RIM01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGCON-SIOCON-12365,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-ADCPSL003,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-ADCPSL-23497,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG040,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-10216,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG041,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11609,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG042,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11637,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG043,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11646,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG044,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12587,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG045,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12309,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG046,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11593,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG047,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-11535,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOG048,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOG-12308,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH049,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-11683,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH050,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-10264,,,,,,
+SKQ201920S,,,,GP03FLMA-RIM01-02-CTDMOH051,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-CTDMOH-12389,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-00-SIOENG000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGCON-SIOCON-14876,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-03-DOSTAD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-DOSTAD-00379,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-04-PHSENF000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-PHSENF-P0103,,,,,,
+SKQ201920S,,,,GP03FLMA-RIS01-05-FLORTD000,7,1,2019-09-20T00:00:00,,CGMGP-03FLMA-00007,,CGINS-FLORTD-01122,,,,,,

```

Comment:
Looks good

Reply:
Are the lat lon correct?  Surveyed anchor locations are (decimal minutes):
HYPM-7 50 06.911 N 144 54.967 W
FLMA-7 50 01.362 N 144 21.647 W
FLMB-7 50 22.533 N 144 30.870 W